src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -71,20 +85,49 @@ export class ServiceSelectionScene extends SceneObjectBase<ServiceSelectionScene
       body: new SceneCSSGridLayout({ children: [] }),
       $variables: new SceneVariableSet({
         variables: [
+          // Service search variable
           new CustomConstantVariable({
             name: VAR_SERVICE,
             label: 'Service',
             hide: VariableHide.hideVariable,
             value: '',
             skipUrlSync: true,
           }),
+          // Service name variable
+          new CustomConstantVariable({
+            name: SERVICE_NAME_VAR,
+            label: '',
+            hide: VariableHide.hideLabel,
+            value: SERVICE_NAME,
+            skipUrlSync: true,
+            options: [
+              {
+                value: SERVICE_NAME,
+                label: SERVICE_NAME,
+              },
+              {
+                value: AGGREGATED_SERVICE_NAME,
+                label: AGGREGATED_SERVICE_NAME,
+              },
+            ],
+          }),
         ],
       }),
-      $data: getQueryRunner([buildResourceQuery(`{${SERVICE_NAME}=~\`.*${VAR_SERVICE_EXPR}.*\`}`, 'volume')]),
+      $data: getSceneQueryRunner({
+        queries: [buildResourceQuery(`{${SERVICE_NAME_EXPR}=~\`.*${VAR_SERVICE_EXPR}.*\`}`, 'volume')],
+        runQueriesMode: 'manual',
```

Comment:
are we still waiting on the labels call here? I don't think we need to now that we a) have the feature flag, and b) aren't using `__aggregated_metrics__` in the volume call.

Reply:
Cleaned up now in the latest commit, no more labels!